gui/main.cc

```diff
@@ -52,15 +89,11 @@ int main(int argc, char** argv)
   QApplication app(argc, argv);
   QApplication::setWindowIcon(QIcon(":/images/appicon.png"));
 
-  QString newPath = "PATH=" + QApplication::applicationDirPath() +
-                    QString(pathSeparator) + getenv("PATH");
-  char* newPathEnv = new char[newPath.length() + 1];
-  strcpy(newPathEnv, newPath.toStdString().c_str());
-  putenv(newPathEnv);
+  set_path();
```

Comment:
I wonder if we should quit trying so hard with OS specific code that's so fragile-looking. Can we look in the application dir and be done?